pandas/tests/frame/indexing/test_indexing.py

```diff
@@ -579,6 +596,8 @@ def test_setitem_cast(self, float_frame):
         float_frame["something"] = 2.5
         assert float_frame["something"].dtype == np.float64
 
+    @td.skip_array_manager_invalid_test
```

Comment:
can you comment on why the test is invalid?  (maybe skip_array_manager_invalid_test could be callable with a "reason" keyword)